custom/ilsgateway/tanzania/warehouse_updater.py

```diff
@@ -266,12 +268,15 @@ def default_start_date():
 
 
 def _get_test_locations(domain):
-    from custom.ilsgateway.tasks import ILS_FACILITIES
+    """
+        returns test region and all its children
+    """
+    test_region = SQLLocation.objects.get(domain=domain, external_id=TEST_REGION_ID)
     sql_locations = SQLLocation.objects.filter(
-        domain=domain,
-        external_id__in=ILS_FACILITIES
+        Q(domain=domain) & (Q(parent=test_region) | Q(parent__parent=test_region))
     ).order_by('id').only('location_id')
-    return [Location.get(sql_location.location_id) for sql_location in sql_locations]
+    return [sql_location.couch_location() for sql_location in sql_locations] + \
+           [test_region.couch_location()]
```

Comment:
@kkrampa this would likely be significantly sped up if you get all the couch docs in bulk. i'm guessing it isn't significant compared to the overhead of the general process but feel free to update if you want.

You could do something like (haven't run/tested this):
```
return [Location.wrap(doc) for doc in iter_docs(Location.get_db(), [sql_location.couch_id for sql_location in [sql_locations + [test_region])]
```